bcipy/simulator/demo/README.md

```diff
@@ -0,0 +1,73 @@
+# Simulator Demo
```

Comment:
Given the patterns of other submodules, I would prefer a `demo` submodule to show how to use existing functionality. These methods can be rolled into other location safely and many would be quite useful.